### PR TITLE
Add attributes tab in IdP edit

### PR DESCRIPTION
**Pull request by savindi7**
### Related Issues
- https://github.com/wso2/product-is/issues/14510

### Related PRs
- None

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/extensions/configs/identity-provider.tsx [hunk 1]
```diff
@@ -49,7 +49,23 @@ export const identityProviderConfig: IdentityProviderConfig = {
...
+            const templateMapping = new Map<string, Set<string>>([
+                [
+                    IdentityProviderTabTypes.USER_ATTRIBUTES, new Set([
+                        IdentityProviderManagementConstants.IDP_TEMPLATE_IDS.FACEBOOK,
+                        IdentityProviderManagementConstants.IDP_TEMPLATE_IDS.GOOGLE,
+                        IdentityProviderManagementConstants.IDP_TEMPLATE_IDS.GITHUB,
+                        IdentityProviderManagementConstants.IDP_TEMPLATE_IDS.OIDC
+                    ])
+                ]
+            ]);
```
Comment (thivi) [inline apps/console/src/extensions/configs/identity-provider.tsx:63]: Shall we move this to the constant class?

File: apps/console/src/features/identity-providers/components/settings/attribute-management/uri-attributes-settings.tsx [hunk 2]
```diff
@@ -46,6 +46,10 @@ interface AdvanceAttributeSettingsPropsInterface extends TestableComponentInterf
      * Specifies if the component should only be read-only.
      */
     isReadOnly: boolean;
+        /**
```
Comment (thivi) [inline apps/console/src/features/identity-providers/components/settings/attribute-management/uri-attributes-settings.tsx:49]: There is a formatting issue here

File: apps/console/src/features/identity-providers/components/settings/attribute-management/uri-attributes-settings.tsx [hunk 3]
```diff
@@ -119,17 +124,33 @@ export const UriAttributesSettings: FunctionComponent<AdvanceAttributeSettingsPr
...
+                                    }
+                                >
+                                The attribute that identifies the user at the enterprise identity provider. 
+                                When attributes are configured based on the authentication response of this 
+                                IdP connection, you can use one of them as the subject. Otherwise, the 
+                                default <Code>saml2:Subject</Code> in the SAML response is used as the 
+                                subject attribute.
+                                </Trans>
+                            )
+                            :
```
Comment (thivi) [inline apps/console/src/features/identity-providers/components/settings/attribute-management/uri-attributes-settings.tsx:142]: Shall we follow a format like this?
```
condition
    ? statement
    : statement
```

File: apps/console/src/features/identity-providers/components/settings/attribute-settings.tsx [hunk 4]
```diff
@@ -369,6 +374,7 @@ export const AttributeSettings: FunctionComponent<AttributeSelectionPropsInterfa
                         subjectError={ isSubmitting && !subjectClaimUri }
                         isReadOnly={ isReadOnly }
                         isMappingEmpty={ isEmpty(selectedClaimsWithMapping) }
+                        isSaml= { isSaml }
```
Comment (thivi) [inline apps/console/src/features/identity-providers/components/settings/attribute-settings.tsx:377]:
```suggestion
                        isSaml={ isSaml }
```

**Review by thivi — APPROVED** [review]

File: apps/console/src/extensions/configs/identity-provider.tsx (hunk 1 above)
Comment (savindi7) [reply apps/console/src/extensions/configs/identity-provider.tsx:63]: It is an extension config.

File: apps/console/src/features/identity-providers/components/settings/attribute-settings.tsx (hunk 4 above)
Comment (savindi7) [reply apps/console/src/features/identity-providers/components/settings/attribute-settings.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/3279/commits/0211cfdea4d21643b59fe812ddcab30928cb0fe5

File: apps/console/src/features/identity-providers/components/settings/attribute-management/uri-attributes-settings.tsx (hunk 3 above)
Comment (savindi7) [reply apps/console/src/features/identity-providers/components/settings/attribute-management/uri-attributes-settings.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/3279/commits/0211cfdea4d21643b59fe812ddcab30928cb0fe5

File: apps/console/src/features/identity-providers/components/settings/attribute-management/uri-attributes-settings.tsx (hunk 2 above)
Comment (savindi7) [reply apps/console/src/features/identity-providers/components/settings/attribute-management/uri-attributes-settings.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/3279/commits/0211cfdea4d21643b59fe812ddcab30928cb0fe5

**Pull request merged**